src/sentry/migrations/0491_remove_orgmemmap_unique_constraints.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.28 on 2023-06-21 19:08
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0490_add_is_test_to_org"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+            ALTER TABLE "sentry_organizationmembermapping" DROP CONSTRAINT "sentry_organizationmembe_organization_id_email_66a560fc_uniq";
+            ALTER TABLE "sentry_organizationmembermapping" DROP CONSTRAINT "sentry_organizationmembe_organization_id_user_id_feb6bdf0_uniq";
+            """,
+            reverse_sql="",
+            hints={"tables": ["sentry_organizationmembermapping"]},
+        ),
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+                migrations.AlterUniqueTogether(
+                    name="organizationmembermapping",
+                    unique_together={("organization_id", "organizationmember_id")},
+                ),
+                migrations.AlterIndexTogether(
+                    name="organizationmembermapping",
+                    index_together={("organization_id", "email"), ("organization_id", "user")},
+                ),
```

Comment:
Do these indexes already exist in the schema? Right now they won't be created as you have these changes in a state_operations list.

Reply:
Correct.  That's because the unique_together already has an index; uniqueness is enforced via indexes.

Reply:
So, after running migrations we get

```
sentry=# \d+ sentry_organizationmembermapping
                                                                       Table "public.sentry_organizationmembermapping"
        Column         |           Type           | Collation | Nullable |                           Default                            | Storage  | Compression | Stats target | Description 
-----------------------+--------------------------+-----------+----------+--------------------------------------------------------------+----------+-------------+--------------+-------------
 id                    | bigint                   |           | not null | nextval('sentry_organizationmembermapping_id_seq'::regclass) | plain    |             |              | 
 organization_id       | bigint                   |           | not null |                                                              | plain    |             |              | 
 date_added            | timestamp with time zone |           | not null |                                                              | plain    |             |              | 
 role                  | character varying(32)    |           | not null |                                                              | extended |             |              | 
 email                 | character varying(75)    |           |          |                                                              | extended |             |              | 
 invite_status         | smallint                 |           |          |                                                              | plain    |             |              | 
 inviter_id            | integer                  |           |          |                                                              | plain    |             |              | 
 user_id               | integer                  |           |          |                                                              | plain    |             |              | 
 organizationmember_id | bigint                   |           |          |                                                              | plain    |             |              | 
Indexes:
    "sentry_organizationmembermapping_pkey" PRIMARY KEY, btree (id)
    "sentry_organizationmembe_organization_id_organiza_1597515e_uniq" UNIQUE CONSTRAINT, btree (organization_id, organizationmember_id)
    "sentry_organizationmembermapping_inviter_id_c1dbf818" btree (inviter_id)
    "sentry_organizationmembermapping_organization_id_94506069" btree (organization_id)
    "sentry_organizationmembermapping_organizationmember_id_f878dba6" btree (organizationmember_id)
    "sentry_organizationmembermapping_user_id_3f7be018" btree (user_id)
Check constraints:
    "sentry_organizationmembermapping_invite_status_check" CHECK (invite_status >= 0)
Foreign-key constraints:
    "sentry_organizationm_inviter_id_c1dbf818_fk_auth_user" FOREIGN KEY (inviter_id) REFERENCES auth_user(id) DEFERRABLE INITIALLY DEFERRED
    "sentry_organizationm_user_id_3f7be018_fk_auth_user" FOREIGN KEY (user_id) REFERENCES auth_user(id) DEFERRABLE INITIALLY DEFERRED
Access method: heap
```

So I may be wrong, hmm.

Reply:
Ok, cool, addressed in https://github.com/getsentry/sentry/pull/51389/commits/84ade6b83ec67832f2d2b2f0191f5f06db29ee4e